src/pages/ExcelOpplasting.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { useState } from 'react';
 import 'nav-frontend-tabell-style';
 import { FormContext, useForm } from 'react-hook-form';
-import { useHistory } from 'react-router-dom';
+import {  useHistory } from 'react-router-dom';
```

Comment:
Jeg tror du har glemt å kjøre linteren.

Reply:
Hvordan kjører jeg linteren 👶

Reply:
npm run lint

evt. "./node_modules/.bin/eslint ./src/**/*.ts* --fix" hvis du er i overkant glad i kommandolinja og vil prøve deg på en automatisk fiks for det den rapporterer. Tror IntelliJ har noe enklere for å gjøre det samme.